### PR TITLE
Other/3874 clean up codebase

### PR DESCRIPTION
**Pull request by nikolai-ri**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -12,187 +12,173 @@ https://github.com/typescript-eslint/tslint-to-eslint-config/blob/master/docs/FA
 Happy linting! 💖
```
Comment (nikolai-ri) [inline .eslintrc.js:12]: Automatically created file, using tslint-to-eslint-config

File: .prettierrc [hunk 2]
```diff
@@ -1,7 +1,12 @@
 {
```
Comment (nikolai-ri) [inline .prettierrc:1]: To align with app-odp codebase

File: package.json [hunk 3]
```diff
@@ -1,51 +1,53 @@
 {
```
Comment (nikolai-ri) [inline package.json:1]: Reformatting and version bump

File: source/Catalog/Catalog.ts [hunk 4]
```diff
@@ -1,154 +1,150 @@
-import { Auth } from "./../auth";
```
Comment (nikolai-ri) [inline source/Catalog/Catalog.ts:1]: Reformatting...

File: source/Catalog/Connectors/DataHubClient.ts [hunk 5]
```diff
@@ -1,72 +1,70 @@
-import { Auth } from "../../auth";
-import { ODP_BACKEND_TOKEN_SCOPE, ODP_DATAHUB_GRAPHQL_ENDPOINT } from "../../constants";
```
Comment (nikolai-ri) [inline source/Catalog/Connectors/DataHubClient.ts:2]: Reformatting...

File: source/Catalog/Connectors/DataMeshApiClient.ts [hunk 6]
```diff
@@ -1,117 +1,117 @@
-import log from "loglevel";
```
Comment (nikolai-ri) [inline source/Catalog/Connectors/DataMeshApiClient.ts:1]: Mostly reformatting + some type improvements and error throwing.

File: source/Catalog/Connectors/HardcodedClient.ts [hunk 7]
```diff
@@ -1,71 +1,71 @@
 import {
-	DATA_LAYERS,
```
Comment (nikolai-ri) [inline source/Catalog/Connectors/HardcodedClient.ts:2]: Reformatting...

File: source/auth.ts [hunk 8]
```diff
@@ -1,155 +1,155 @@
 import {
-	AuthenticationResult,
```
Comment (nikolai-ri) [inline source/auth.ts:2]: Reformatting...

File: source/castUtils.ts [hunk 9]
```diff
@@ -1,77 +1,78 @@
-import { IGeoLocation, CastColumnTypeEnum } from "./types";
```
Comment (nikolai-ri) [inline source/castUtils.ts:1]: Reformatting and type improvements. This will be deprecated in the next version.

File: source/casts.ts [hunk 10]
```diff
@@ -1,20 +1,21 @@
-import { ExternalId, FileLink, Sequence, SequenceListScope, SequenceRowsRetrieve } from "@cognite/sdk";
```
Comment (nikolai-ri) [inline source/casts.ts:1]: Reformatting and type improvements. This will be deprecated in the next version.

File: source/geoUtils.ts [hunk 11]
```diff
@@ -1,80 +1,80 @@
-import * as https from "https";
```
Comment (nikolai-ri) [inline source/geoUtils.ts:1]: Reformatting and type improvements. This will be deprecated in the next version.

File: source/marineRegions.ts [hunk 12]
```diff
@@ -1,185 +1,189 @@
-import { Sequence } from "@cognite/sdk";
```
Comment (nikolai-ri) [inline source/marineRegions.ts:1]: Reformatting and type improvements. This will be deprecated in the next version.

File: source/types.ts [hunk 13]
```diff
@@ -1,438 +1,436 @@
 import {
-	Aggregate,
```
Comment (nikolai-ri) [inline source/types.ts:2]: Reformatting and some minor type changes.

File: tsconfig.json [hunk 14]
```diff
@@ -1,19 +1,18 @@
 {
-	"compilerOptions": {
```
Comment (nikolai-ri) [inline tsconfig.json:2]: More alignment with app-odp.

**Review by albertsmw — APPROVED** [review]
Looks good :). Thanks for the reformatting comments so I wouldn't have to read everything :).

**Pull request merged**